### DISCUSSION TITLE
Include pagebreaks when converting to Markdown

### DISCUSSION THREAD

**Discussion by tbdalgaard** [jgm/pandoc#9545] [Q&A]
Hi!

I have found different filters that allow me to insert pagebreaks into Markdown documents. What I want is a way to get this information when i convert from Latex, DOCX etc. to Markdown.

Could this be integrated into Pandoc itself or is there another way to get this information wen converting from one format to Markdown?

Thanks in advance.

Best regards Thomas

**Comment by tarleb** [comment]
See the discussion in #1934.
